### ISSUE TITLE
How to force refresh when switching clusters on dashboard?

### ISSUE THREAD

**Issue by TomJohnson-Syncbak** [trevorndodds/elasticsearch-metrics#33]
First, thanks for a great dashboard!

I augmented the Python data gathering script and then rewrote it in Go (for speed and to follow internal preference for our Elasticsearch tools).  We wanted to use one instance of the tool to collect data on two or more clusters.

I'm now feeding stats from 3 clusters into the Grafana ES instance.  When I change the cluster by using the dropdown menu, I cannot seem to make the panels on the dashboard refresh.

**Comment by trevorndodds** [comment]
It should automatically refresh when you select the different Cluster.

The variables Cluster / Node are set to refresh on dashboard load too. That should update the graphs.

**Comment by TomJohnson-Syncbak** [comment]
I checked to make sure those variables are still set to refresh on dashboard load.  They are.

My cluster names are formatted as "dddddddddddd:x-y-z" where d is a decimal digit and x, y, and z are lowercase letters.  Perhaps that is the problem?

**Comment by TomJohnson-Syncbak** [comment]
Sorry, I ended up aliasing my cluster names to remove characters that Elasticsearch treats as separators.  It's easier for me to do that rather than remember to set the field to be "not_analyzed".

Not the dashboard's fault.

One comment, I found the juxtaposition of this advice on the dashboard confusing:

- You no longer need to do this, rather set the template variable to name.raw (v2) or name.keyword (v5) same applies for cluster_name
- If your cluster_name or node names have "-" you will have to load a custom index to set the "name" field to not_analyzed:

It's unclear what "this" you are not required to do.
